app/build.gradle

```diff
@@ -1,13 +1,14 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 23
-    buildToolsVersion "23.0.1"
+    compileSdkVersion 27
+    buildToolsVersion '28.0.2'
 
     defaultConfig {
-        applicationId "com.ichi2.apisample"
-        minSdkVersion 15
-        targetSdkVersion 23
+        applicationId "com.github.vi.AnkiImporter"
+        // Setting minSdkVersion to 20 to enable Instant Run.
+        minSdkVersion 20
```

Comment:
So it won't work on my own device?

Is there some real reason to limit to  Android >5.0?

Reply:
Ah - nothing beyond enabling "Instant Run". You can undo that minSdkVersion change as you merge.